README.md

```diff
@@ -271,25 +270,46 @@ See [these](https://github.com/JuliaRegistries/General/pull/31549#issuecomment-7
+If there is some urgent need for the package to be registered (for example, an upcoming talk featuring the package or other external deadline), the waiting period can be expedited by request made in the [#pkg-registration](https://julialang.slack.com/archives/C6M4DQA5P) channel on the
 public Julia Language Slack, or directly on the PR itself. This request should include a link to the registration PR and why an expedited merge is needed. In this case, a registry maintainer (see below) can manually review the package (often with stricter standards than the typical package faces, given the lack of bandwidth to review all packages thoroughly).
 
-#### Who can approve an early merge?
+### Who can approve an early merge?
 
 Any registry maintainer may merge a package registration PR early.
 Early merges should still be discussed on the [#pkg-registration](https://julialang.slack.com/archives/C6M4DQA5P) slack channel before merging.
 Registry maintainers are discouraged from merging packages they directly *or* indirectly maintain as mentioned in [CONTRIBUTING.md](https://github.com/JuliaRegistries/General/blob/master/CONTRIBUTING.md#other-ways-to-help).
 
+### When is yanking a release appropriate?
+
+Releases that have already merged can be marked as "yanked" by adding `yanked = true` under the release
+entry in `Versions.toml` ([example](https://github.com/JuliaRegistries/General/pull/94858/files)). This
+tells Pkg to no longer resolve those versions (e.g. when running `Pkg.add()` or `Pkg.resolve()`). However,
+to maintain perfect reproducibility of environments specified with a full manifest, yanked versions that
+are specifically listed in a manifest will continue to instantiate when running `Pkg.instantiate()` and
+yanked versions cannot be re-registered in General with different source code.
+
+The primary reason yanking exists is to make a release impossible to install. This is intended to be used for circumstances when having this
+release installed poses a risk to security or safety. It is not intended for avoiding people installing releases with bugs, for that use a
+patch release -- potentially after doing nothing more than `git revert`  in the package repo. All patch releases prior to the most recent have known bugs by definition.
+
+There is however, a special category of bugged releases that can not be resolved by having a patch release. These also need to be resolved by
+yanking. That special category is when the compat bounds have been set too wide. i.e. say `v2.10.0` was released using a feature not on julia
+`v1.6` but the compat entry for julia was not raised in the release. In this case releasing a `v2.10.1` with the corrected julia compat would
+not solve the issue as on julia v1.6 Pkg would still resolve the broken `v2.10.0`, and as a minor bump, reverting the code changes would not
+be valid in a patch bump.
```

Comment:
Could this sort of too-wide compat be resolved by making a PR to the registry with the correct bounds instead of yanking? After all, the release is (a priori) good, just not for all originally specified versions.